[libcurl]  Link errors using with_winssl option

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **libcurl/7.64.1**
  * Operating System+version: **Windows**
  * Compiler+version: **MSVC 2017**
  * Conan version: **conan 1.16.1**
  * Python version: **Python 3.7.0**

### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
04/07/2019 14:03:26: Configuration:
04/07/2019 14:03:26: [settings]
04/07/2019 14:03:26: arch=x86_64
04/07/2019 14:03:26: arch_build=x86_64
04/07/2019 14:03:26: build_type=Debug
04/07/2019 14:03:26: compiler=Visual Studio
04/07/2019 14:03:26: compiler.runtime=MDd
04/07/2019 14:03:26: compiler.toolset=v141
04/07/2019 14:03:26: compiler.version=15
04/07/2019 14:03:26: os=Windows
04/07/2019 14:03:26: os_build=Windows
04/07/2019 14:03:26: [options]
04/07/2019 14:03:26: [build_requires]
04/07/2019 14:03:26: [env]
```

### Steps to reproduce (Include if Applicable)
Using conan via Visual Studio Conan extension with following conanfile.txt:
```
[requires]
libcurl/7.64.1@bincrafters/stable

[options]
libcurl:with_openssl=False
libcurl:with_winssl=True

[generators]
visual_studio
```

Try to use libcurl, for example build this:
```
#include <stdio.h>
#include <curl/curl.h>
 
int main(void)
{
  CURL *curl;
  CURLcode res;
 
  curl = curl_easy_init();
  if(curl) {
    curl_easy_setopt(curl, CURLOPT_URL, "https://example.com");
    /* example.com is redirected, so we tell libcurl to follow redirection */ 
    curl_easy_setopt(curl, CURLOPT_FOLLOWLOCATION, 1L);
 
    /* Perform the request, res will get the return code */ 
    res = curl_easy_perform(curl);
    /* Check for errors */ 
    if(res != CURLE_OK)
      fprintf(stderr, "curl_easy_perform() failed: %s\n",
              curl_easy_strerror(res));
 
    /* always cleanup */ 
    curl_easy_cleanup(curl);
  }
  return 0;
}
```

Gets link errors.

### Logs (Include/Attach if Applicable)
```
1>libcurl.lib(schannel.obj) : error LNK2019: unresolved external symbol __imp_CertOpenStore referenced in function schannel_connect_step1
1>libcurl.lib(schannel_verify.obj) : error LNK2001: unresolved external symbol __imp_CertOpenStore
1>libcurl.lib(schannel.obj) : error LNK2019: unresolved external symbol __imp_CertCloseStore referenced in function schannel_connect_step1
1>libcurl.lib(schannel_verify.obj) : error LNK2001: unresolved external symbol __imp_CertCloseStore
1>libcurl.lib(schannel.obj) : error LNK2019: unresolved external symbol __imp_CertFindCertificateInStore referenced in function schannel_connect_step1
1>libcurl.lib(schannel.obj) : error LNK2019: unresolved external symbol __imp_CertFreeCertificateContext referenced in function pkp_pin_peer_pubkey
1>libcurl.lib(schannel_verify.obj) : error LNK2001: unresolved external symbol __imp_CertFreeCertificateContext
1>libcurl.lib(schannel.obj) : error LNK2019: unresolved external symbol __imp_CryptStringToBinaryA referenced in function schannel_connect_step1
1>libcurl.lib(schannel_verify.obj) : error LNK2019: unresolved external symbol __imp_CertAddCertificateContextToStore referenced in function add_certs_to_store
1>libcurl.lib(schannel_verify.obj) : error LNK2019: unresolved external symbol __imp_CertGetNameStringA referenced in function verify_host
1>libcurl.lib(schannel_verify.obj) : error LNK2019: unresolved external symbol __imp_CryptQueryObject referenced in function add_certs_to_store
1>libcurl.lib(schannel_verify.obj) : error LNK2019: unresolved external symbol __imp_CertCreateCertificateChainEngine referenced in function verify_certificate
1>libcurl.lib(schannel_verify.obj) : error LNK2019: unresolved external symbol __imp_CertFreeCertificateChainEngine referenced in function verify_certificate
1>libcurl.lib(schannel_verify.obj) : error LNK2019: unresolved external symbol __imp_CertGetCertificateChain referenced in function verify_certificate
1>libcurl.lib(schannel_verify.obj) : error LNK2019: unresolved external symbol __imp_CertFreeCertificateChain referenced in function verify_certificate
```

--- a/conanfile.py
+++ b/conanfile.py
@@ -413,6 +413,8 @@ class LibcurlConan(ConanFile):
             self.cpp_info.libs.append('ws2_32')
             if self.options.with_ldap:
                 self.cpp_info.libs.append("wldap32")
+            if self.options.with_winssl:
+                self.cpp_info.libs.append("Crypt32")
 
         if self.is_mingw:
             # provide pthread for dependent packages

